src/Mvc/Mvc.Testing/src/buildTransitive/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -1,5 +1,5 @@
 <Project>
 
-  <Import Project="..\..\build\$(TargetFramework)\Microsoft.AspNetCore.Mvc.Testing.targets" />
+  <Import Project="..\..\build\net6.0\Microsoft.AspNetCore.Mvc.Testing.targets" />
```

Comment:
FYI @wtgodbe this is going in main, but would be affected by the net7.0 TFM change.

Reply:
Cool if I just fold this in to that PR?

Reply:
Or do you need it immediately?

Reply:
I was looking to backport this to 6.0 to address this: https://github.com/dotnet/aspnetcore/issues/36510

Reply:
Go for it - I'll make a conflicting change in the other PR to remind myself